app/views/layouts/application.html.erb

```diff
@@ -20,6 +20,10 @@
   <%= yield %>
 </div>
 
+
+<%= react_component "Footer" %>
+
+
```

Comment:
I am amazed that our linter doesn't care about multiple blank lines. So I don't make it a blocker. To me, this PR is ready to be merged.

Good job @Yassa-hue 👍🏾